.circleci/config.yml

```diff
@@ -104,6 +104,18 @@ jobs:
           path: test-results
           destination: test-results
 
+  publish:
```

Comment:
**Question**: why is this new step required?

Reply:
this step is meant to publish the `n-syndication` package on npm